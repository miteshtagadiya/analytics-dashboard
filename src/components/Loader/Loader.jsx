import React from "react";

const Loader = () => {
  return (
    <div
      className="d-flex justify-content-center align-items-center"
      style={{ minHeight: 270 }}
    >
      <div className="spinner-grow" role="status">
        <span className="sr-only">Loading...</span>
      </div>
    </div>
  );
};

export default Loader;
