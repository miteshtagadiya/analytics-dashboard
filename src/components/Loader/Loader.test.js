import React from "react";
import renderer from "react-test-renderer";
import Loader from "./Loader";

test("renders correctly", () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
