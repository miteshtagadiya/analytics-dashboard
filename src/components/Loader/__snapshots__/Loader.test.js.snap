// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="d-flex justify-content-center align-items-center"
  style={
    Object {
      "minHeight": 270,
    }
  }
>
  <div
    className="spinner-grow"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
